tests/test_accessory.py

```diff
@@ -1,58 +1,71 @@
 """Tests for pyhap.accessory."""
+from unittest.mock import patch
+
 import pytest
 
-import pyhap.accessory as accessory
-import pyhap.loader as loader
+from pyhap.accessory import Accessory, Bridge
+from pyhap.accessory_driver import AccessoryDriver
+from pyhap.const import STANDALONE_AID
+
+with patch('pyhap.accessory_driver.HAPServer'), \
+        patch('pyhap.accessory_driver.Zeroconf'):
+    DRIVER = AccessoryDriver()
+
+
+# #### Accessory ####
+# ###################
+
+def test_init():
+    Accessory(DRIVER, 'Test Accessory')
 
 
-class TestAccessory:
```

Comment:
Why are we removing the classes? This prevents you from running tests only from a single test class. Is the performance issue really big - there only a handful of methods?